### PR TITLE
Specify support for DI suites for Spruce

### PR DESCRIPTION
**Pull request by sbihel**

### CODE REVIEW

**Review by aljones15 — APPROVED** [review]
The ed25519 issuer works so approving pending verification that the eddsa and ecdsa endpoints are functioning.

File: implementations/Spruce.json [hunk 1]
```diff
@@ -7,12 +7,34 @@
...
+    "id": "did:key:z6MkgND5U5Kedizov5nxeh2ZCVUTDRSmAfbNqPhzCq8b72Ra",
+    "endpoint": "https://vc.spruceid.xyz/credentials/issue",
+    "tags": ["vc-api", "JWT"]
+  }, {
+    "id": "did:key:z6MkgND5U5Kedizov5nxeh2ZCVUTDRSmAfbNqPhzCq8b72Ra",
+    "endpoint": "https://vc.spruceid.xyz/credentials/issue",
+    "options": {
+      "type": "DataIntegrityProof"
+    },
+    "tags": ["vc-api", "eddsa-2020"]
```
[the remark this answers is not in the document]
Comment (JSAssassin) [reply implementations/Spruce.json:21]: or is `eddsa-2020` the same as `Ed25519Signature2020`?

**Review by JSAssassin — APPROVED** [review]
Left some comments but approving since the endpoints seems to be operational.

File: implementations/Spruce.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (sbihel) [reply implementations/Spruce.json]: It was just a typo, sorry for the confusion

File: implementations/Spruce.json [hunk 2]
```diff
@@ -7,12 +7,34 @@
...
+  }, {
+    "id": "did:key:zDnaei6w9La4VphyPA6V2V5poBTpVtjCsnzPML5c78GWjRhnc",
+    "endpoint": "https://vc.spruceid.xyz/credentials/issue",
+    "options": {
+      "type": "DataIntegrityProof"
+    },
+    "tags": ["vc-api", "ecdsa-2019"]
```
[the remark this answers is not in the document]
Comment (sbihel) [reply implementations/Spruce.json]: Little mix up in the deployed version, it is fixed now

[the remark this answers is not in the document]
Comment (JSAssassin) [reply implementations/Spruce.json]: @sbihel Thank you for fixing that, I can confirm that the ecdsa tests are passing for Spruce now. 
`eddsa-2022` issuer is giving a 500 though, whenever you get a chance, can you also please check that one. Thanks.

[the remark this answers is not in the document]
Comment (sbihel) [reply implementations/Spruce.json]: Apologies, this is fixed now

[the remark this answers is not in the document]
Comment (JSAssassin) [reply implementations/Spruce.json]: @sbihel Perfect! Thank you, I can confirm it's fixed. I'll get this PR merged now.

**Pull request merged**